daemon/src/db.rs

```diff
@@ -246,159 +232,205 @@ pub async fn load_cfd_by_order_id(
     order_id: OrderId,
     conn: &mut PoolConnection<Sqlite>,
 ) -> Result<Cfd> {
-    let order_uuid = serde_json::to_string(&order_id)?;
-
     let row = sqlx::query!(
         r#"
-        select
-            orders.uuid as order_id,
-            orders.initial_price as price,
-            orders.min_quantity as min_quantity,
-            orders.max_quantity as max_quantity,
-            orders.leverage as leverage,
-            orders.trading_pair as trading_pair,
-            orders.position as position,
-            orders.origin as origin,
-            orders.liquidation_price as liquidation_price,
-            orders.creation_timestamp as creation_timestamp,
-            orders.term as term,
-            orders.oracle_event_id,
-            cfds.quantity_usd as quantity_usd,
-            cfd_states.state as state
-        from cfds as cfds
-        inner join orders as orders on cfds.order_id = orders.id
-        inner join cfd_states as cfd_states on cfd_states.cfd_id = cfds.id
-        where cfd_states.state in (
+        with ord as (
```

Comment:
nit: could we name these placeholders like the tables that they represent? 
I find it hard to map from table to placeholder to next statement (...).